registry/registry-polaris/pom.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.alipay.sofa</groupId>
+        <artifactId>sofa-rpc-registry</artifactId>
+        <version>${revision}</version>
+    </parent>
+    <artifactId>sofa-rpc-registry-polaris</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>com.alipay.sofa</groupId>
+            <artifactId>sofa-rpc-log</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.alipay.sofa</groupId>
+            <artifactId>sofa-rpc-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-netty-shaded</artifactId>
+            <version>1.33.0</version>
```

Comment:
the package `grpc-netty-shaded` must be specified, otherwise  `sofa-rpc`  and ` polaris` will conflict because of this package.

Reply:
Because of the special package method in sofarpc, sofa-rpc only provides one jar when use. This `polaris` module code will merge into `sofa-rpc-all` jar when released, so the final version of `grpc-netty-shaded` will depend on the `all` module version, let's see if we can upgrade the `grpc-netty-shaded` version  in global.

Reply:
so I will specify the version of this package  at this module